analyzer/src/main/kotlin/PackageManager.kt

```diff
@@ -130,7 +130,16 @@ abstract class PackageManager(
                             // That is, at the example of a PIP project, if a directory contains all three files
                             // "requirements-py2.txt", "requirements-py3.txt" and "setup.py", only consider the
                             // former two as they match the glob with the highest priority, but ignore "setup.py".
-                            result.getOrPut(manager) { mutableListOf() } += matchesPerGlob.first()
+                            val priorityMatches = matchesPerGlob.first()
```

Comment:
We currently have the problem that we do no support Gradle projects which define multiple projects in a single `build.gradle`. The only workaround I know for such projects is to put empty `build.gradle` files into the directories of the subprojects. I didn't test it but I expect that this change would make the workaround impossible.

Reply:
Wouldn't it be good practice to put a comment into those Gradle files then anyway, might makes them non-empty, and it would be working again?